devito/finite_differences/derivative.py

```diff
@@ -40,6 +39,8 @@ class Derivative(sympy.Derivative, Differentiable):
     transpose : Transpose, optional
         Forward (matvec=direct) or transpose (matvec=transpose) mode of the
         finite difference. Defaults to ``direct``.
+    x0 : Dict, optional
```

Comment:
dict, not Dict

"Dictionary" redundant in the description. I also think the description could be made clearer.  What exactly u mean by "scheme origin" (note: I do understand it now, but for a random reader...)